packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -111,18 +111,21 @@ export default class Dropdown extends Component {
       ...props
     } = this.props
 
+    const isVisible = this.state.visible
```

Comment:
could also just destructure this and use `visible ? 'expand' ... ` ? but nbd

Reply:
There is already one called `visible` from props.